### DISCUSSION TITLE
number of virtual desktop

### DISCUSSION THREAD

**Discussion by ivanottantuno** [WayfireWM/wayfire#1101] [Q&A] [answered]
Hi,
it's possible to have more than 9 virtual desktop?
Tnx
Ivan

**Comment by ammen99** [comment, marked as answer]
Yes, it is. Check the `core/vwidth` and `core/vheight` options (you can set them to the number of columns/rows of workspaces)

**Reply by ivanottantuno** [reply]
Tnx a lot
